src/components/flow/routers/subflow/SubflowRouterForm.tsx

```diff
@@ -89,7 +89,9 @@ export default class SubflowRouterForm extends React.PureComponent<
     );
 
     if (this.state.valid && !hasFieldErrors) {
-      this.props.updateRouter(stateToNode(this.props.nodeSettings, this.state));
+      this.props.updateRouter(
+        stateToNode(this.props.nodeSettings, this.state, this.context.config.flowType)
```

Comment:
during handleSave, when i tried using `this.context.config.definition.type` i was getting an error that definition is undefined, but when i set a breakpoint, i noticed that `this.context.config.flowType` contains what i believe is the parent flow type - it's a bit unexpected as i thought the idea was to get the flow type from the definition, so here's some more context...

during handleSave, `this.context.config.flow` = `e19aa222-2ca8-4328-a16a-2a9f63911b9f` which matches the flow uuid value in the URL = `e19aa222-2ca8-4328-a16a-2a9f63911b9f`, which i assume is the parent flow's uuid, whereas `this.state.flow.value.uuid` = `cacddc3e-9393-4303-af6c-447b089bb494` which i'm assuming is the subflow's uuid...

and so `this.context.config.flowType` will get the job done, but i just wanted to do a final sanity check to make sure this is the correct arg to pass into stateToNode